src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -160,14 +159,18 @@ public Task RunAsync()
                     ColorConsole.WriteLine(new ColorToken[] { padding }.Concat(line).ToArray());
                 }
 
+                ColorConsole.WriteLine();
+
+                var totalColor = currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0];
+                ColorConsole.Write(padding, $"{currentLeaderboard?.Count ?? 0:N0} ".Color(totalColor), $"{itemsName}".DarkGray());
+
                 if (currentLeaderboard.Since.HasValue)
                 {
                     ColorConsole.WriteLine(
-                        padding,
-                        $"{currentLeaderboard.Since?.ToLocalTime()} to {currentLeaderboard.LastActivityDateTime?.ToLocalTime()}".Gray());
+                        $" since ".DarkGray(),
+                        $"{currentLeaderboard.Since?.ToLocalTime():dddd} {currentLeaderboard.Since?.ToLocalTime():hh:mm}".Gray());

```

Comment:
`HH` gives 24 hour time.
`hh` gives 12 hour time.

Given that we don't show AM/PM, I'd use 24 hour time

Reply:
Oops, good catch! Will fix.